_docs/principles/accessibility.md

```diff
@@ -3,29 +3,67 @@ title: Accessibility
 layout: docs
 category: Principles
 usage: >
-  Visual accessibility standards enable users with varied levels of color impairment to successfully navigate and use a Helix-based UI.
+  Accessibility standards promote the design of user interfaces (UI) so they can be used by people with the widest range of abilities within the widest range of situations.
 status: stable
 resource: true
-last-modified: 2017-10-24
+last-modified: 2019-04-25
 ---
 
 {% include toc.html %}
 
 <section class="static-section" markdown="1">
 
-## Accessibility for visual design
-
 <div class="hxRow"  markdown="1">
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-12-md hxSpan-12-lg" %}
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Understanding accessibility
+
+An accessible UI is designed so that it can be *accessed* and used by people who suffer from a permanent or a temporary impairment. While visual impairments such as color blindness and lack of sight are two of the most obvious impairments to consider, an impairment can also be auditory, physical, or cognitive. For example, a user who suffers from a concussion may experience difficulty processing information, so ensure that you write succinct on-page instructions and labels. A user who experiences hand tremors as a side effect to medication has difficulty operating a mouse, so design pointer targets so they are large enough.
+
+Designing for accessibility is best practice because you consider all users and not just those with a permanent disability.
```

Comment:
Designing for accessibility is a best practice when using Helix because your designs will be usable by everyone regardless of whether they have any impairments.

Reply:
Done.